src/utils/recursion.js

```diff
@@ -0,0 +1,29 @@
+import _ from 'lodash';
+
+function mapKeysDeep(object, callback) {
```

Comment:
[MISC] Could you add a little comment here to explain what this function does?

Reply:
Plus another comment to clarify the Symbols part (other might come with the same question as mine)